src/pe_reports/report_generator.py

```diff
@@ -22,21 +44,73 @@ def load_template():
 
 def export_set(prs):
     """Export PowerPoint report set to output directory."""
-    pptx_out = "Customer_ID_Posture_Exposure.pptx"
-    prs.save(os.path.join(SAVE_PATH, pptx_out))
+    try:
+        pptx_out = "Customer_ID_Posture_Exposure.pptx"
+        prs.save(os.path.join("/output", pptx_out))
```

Comment:
Why aren't you using the `OUTPUT_DIRECTORY` argument here?

Reply:
Resolved in 4d64c0f